audit_log/tests/runtests.py

```diff
@@ -8,68 +8,75 @@
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "django.contrib.messages",
+    "django.contrib.staticfiles",
+]
 
-ALWAYS_MIDDLEWARE_CLASSES = (
-    'django.contrib.sessions.middleware.SessionMiddleware',
-    'django.middleware.common.CommonMiddleware',
-    'django.middleware.csrf.CsrfViewMiddleware',
-    'django.contrib.auth.middleware.AuthenticationMiddleware',
-    'django.contrib.messages.middleware.MessageMiddleware',
-    'django.middleware.clickjacking.XFrameOptionsMiddleware',
-    'audit_log.middleware.UserLoggingMiddleware',
-)
+ALWAYS_MIDDLEWARE_CLASSES = [
+    "django.contrib.sessions.middleware.SessionMiddleware",
+    "django.middleware.common.CommonMiddleware",
+    "django.middleware.csrf.CsrfViewMiddleware",
+    "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "django.contrib.messages.middleware.MessageMiddleware",
+    "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "audit_log.middleware.UserLoggingMiddleware",
+]
 
 
 settings.configure(
     SECRET_KEY="django_tests_secret_key",
     DEBUG=True,
     TEMPLATE_DEBUG=True,
-    ALLOWED_HOSTS=[],
+    TEMPLATES=[
+        {
+            "BACKEND": "django.template.backends.django.DjangoTemplates",
+            "DIRS": [],
+            "APP_DIRS": True,
+            "OPTIONS": {
+                "context_processors": [
+                    "django.template.context_processors.debug",
+                    "django.template.context_processors.request",
+                    "django.contrib.auth.context_processors.auth",
+                    "django.contrib.messages.context_processors.messages",
+                ],
+            },
+        },
+    ],
+    ALLOWED_HOSTS=["*"],
     INSTALLED_APPS=ALWAYS_INSTALLED_APPS + CUSTOM_INSTALLED_APPS,
-    MIDDLEWARE_CLASSES=ALWAYS_MIDDLEWARE_CLASSES,
-    ROOT_URLCONF='tests.urls',
+    MIDDLEWARE=ALWAYS_MIDDLEWARE_CLASSES,
+    ROOT_URLCONF="tests.urls",
     DATABASES={
-        'default': {
-            'ENGINE': 'django.db.backends.sqlite3',
+        "default": {
+            "ENGINE": "django.db.backends.sqlite3",
         }
     },
-    LANGUAGE_CODE='en-us',
-    TIME_ZONE='UTC',
+    LANGUAGE_CODE="en-us",
+    TIME_ZONE="UTC",
     USE_I18N=True,
     USE_L10N=True,
     USE_TZ=True,
-    STATIC_URL='/static/',
+    STATIC_URL="/static/",
     # Use a fast hasher to speed up tests.
-    PASSWORD_HASHERS=(
-        'django.contrib.auth.hashers.MD5PasswordHasher',
-    ),
-    FIXTURE_DIRS=glob.glob(BASE_DIR + '/' + '*/fixtures/'),
-    TEMPLATE_DIRS = (
-        os.path.abspath(os.path.join(BASE_DIR, 'templates')),
-    ),
-
-
+    PASSWORD_HASHERS=("django.contrib.auth.hashers.MD5PasswordHasher",),
+    TEMPLATE_DIRS=(os.path.abspath(os.path.join(BASE_DIR, "templates")),),
 )
 
 django.setup()
-args = [sys.argv[0], 'test']
+args = [sys.argv[0], "test"]
 # Current module (``tests``) and its submodules.
-test_cases = '.'
+test_cases = "."
```

Comment:
Propose to not introduce any code-style refactoring in this PR - use single quotes